src/popup/router/pages/Notifications.vue

```diff
@@ -65,7 +65,8 @@ export default {
               .map((s) => s.type)
               .includes(n.type),
         )
-        .sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt));
+        .sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt))
+        .map((n) => (n.chainName ? { ...n, chainName: n.chainName?.[0] } : n));
```

Comment:
```suggestion
        .map((n) => (n.chainName ? { ...n, chainName: n.chainName.[0] } : n));
```
as I understand, the second check of `chainName` is an extra one, but I'm fine to merge it as it is